apache-maven/src/main/appended-resources/META-INF/LICENSE.vm

```diff
@@ -22,21 +22,18 @@ Apache Maven includes a number of components and libraries with separate
 copyright notices and license terms. Your use of those components are 
 subject to the terms and conditions of the following licenses: 
 ##
-#set ( $apacheLicenseNames = [ "Apache License, Version 2.0", "The Apache Software License, Version 2.0",
-    "ASLv2", "Apache Public License 2.0", "Apache 2.0" ] )
+#set ( $apacheMavenGroupIds = [ "org.apache.maven", "org.apache.maven.wagon", "org.apache.maven.resolver",
+                                "org.apache.maven.shared" ] )
 #set ( $MITLicenseNames = [ "MIT License", "MIT license", "The MIT License" ] )
 #foreach ( $project in $projects )
 #**##foreach ( $license in $project.licenses)
-#*  *##if ( !$apacheLicenseNames.contains( $license.name ) )
+#*  *##set ( $groupId = $project.artifact.groupId )
+#*  *##if ( !$apacheMavenGroupIds.contains( $groupId ) )
 #*    *##set ( $artId = $project.artifact.artifactId )
 #*    *##set ( $url = $license.url )
 #*    *##set ( $spdx = false )
 #*    *##set ( $includeLicense = true )
 #*    *###
-#*    *##if ( $project.artifact.artifactId == "jcl-over-slf4j" )
```

Comment:
this is to be discussed, if we should really report a different license other the one that is reported on Maven Central

Reply:
This must be raised upstream.

Reply:
everything already done: license in pom.xml was not consistent with license in scm
see https://issues.apache.org/jira/browse/MNG-6779 and the associated PR to slf4j that has already been merged

Reply:
This means we have to wait for 1.7.29, don't we?

Reply:
we need to wait for a fixed release to remove the hack that overrides the license: the hack is not so big, we can live with it for now
(notice the PR https://github.com/qos-ch/slf4j/pull/226 was merged in 2.0.0-alpha branch only)

Reply:
That's bad and needs to be backported to 1.7.x line.

Reply:
The real problem is that we can change here the output in LICENSE but we can't change the license file we download to 'lib'.
So they won't be consistent